csharp-basic-topics/ValueAndReferenceTypes/ValueAndReferenceTypes/ReferenceTypeSample.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ValueAndReferenceTypes
+{
+    public class ReferenceTypeRectangle
+    {
+        private int _length = 50;
+        private int _breadth = 20;
+        public int length { 
+            get { return _length; } 
+            set { _length = value; }
+        }
+        public int breadth
+        {
+            get { return _breadth; }
+            set { _breadth = value; }
+        }     
```

Comment:
These properties are not useful since they just return the backed field. hey should be removed.